modules/getting-started/proc-customize-rhdh-homepage.adoc

```diff
@@ -1,9 +1,22 @@
 [id='proc-customize-rhdh-homepage_{context}']
 = Customizing the Home page in {product}
 
-To access the Home page in {product}, the base URL must include the `/developer-hub` proxy. You can configure the Home page by passing the data into the `app-config.yaml` file as a proxy. You can provide data to the Home page by using the following sources:
+To access the Home page in {product}, the base URL must include the `/developer-hub` proxy. You can configure the Home page by passing the data into the `app-config.yaml` file as a proxy. You can provide data to the Home page from the following sources:
 
-* JSON files hosted on GitHub or GitLab. To access the data from the JSON files, add the following code to the `app-config.yaml` file:
+* JSON files hosted on GitHub or GitLab.
+* A dedicated service that provides the Home page data in JSON format using an API.
+
+== Using hosted JSON files to provide data to the Learning Paths
+
+.Prerequisites
+
+You have installed {product} by using either the Operator or Helm chart. For more information, see xref:proc-install-rhdh_rhdh-getting-started[].
```

Comment:
[SME] can you confirm that using the JSON files works regardless of whether RHDH is installed with Operator or Helm?

Reply:
yeah that's correct